maestro-test/src/test/kotlin/maestro/test/IntegrationTest.kt

```diff
@@ -612,75 +602,6 @@ class IntegrationTest {
         }
     }
 
-    @Test
-    fun `Case 023 - runFlow with initFlow`() {
```

Comment:
ditto, keeping one test at least which asserts the correct exception would be nice until we fully deprecate it

Reply:
Done ✅